### ISSUE TITLE
Suggestion: rel=me support

### ISSUE THREAD

**Issue by techmagus** [PositiveSumNet/SocialAssistant#9]
Hi,

When I check a profile page, like mine (https://whosum.com/gh/techmagus), I noticed that there is no `rel="me"` in the links or in the page itself.

For services using it for detecting relationships, like IndieWeb and the fediverse software like Mastodon, to mention two, doesn't know the relationship. For example, in a fediverse software like Mastodon, it adds a checkmark to the link if it detects `rel=me` (for example my profile: https://c.im/@youronlyone )

I would like to suggest `rel="me"` support in the profile pages as well as the account links.

Thank you!

**Comment by scafaria** [comment]
Hi. We just published a new feature. If you instead give Mastodon https://possumid.com/gh/techmagus it will verify. The issue was that whosum is generated via javascript but Mastodon does not support that. Possumid.com on the other hand is recognized.

**Comment by techmagus** [comment]
Hello,

That was quick! I used the possumid URL, and it works now.

Thank you very much!

PS
Not sure though why a redirect URL is not working, it's working on the other service. I don't recall it being a feature of the other service, it's only something I tried and it worked. In any case, it is outside the scope of your project (probably something in my redirect settings). ^_^

Update:
Just for reference: the URL changed, I misunderstood your reply. With my URL redirect pointing to the new URL, it works perfectly.
